install/pre-installation.md

```diff
@@ -409,6 +402,42 @@ in `/etc/environment` from the [Download CSM tarball](#21-download-csm-tarball)
       install -y csm-testing goss-servers
    ```
 
+1. (`pit#`) Get the artifact versions.
+
+   ```bash
+   kubernetes_version="$(find ${CSM_PATH}/images/kubernetes -name '*.squashfs' -exec basename {} .squashfs \; | awk -F '-' '{print $NF}')"
+   ceph_version="$(find ${CSM_PATH}/images/storage-ceph -name '*.squashfs' -exec basename {} .squashfs \; | awk -F '-' '{print $NF}')"
+   ```
+
+1. (`pit#`) Copy the NCN images from the expanded tarball.
+
+   > ***NOTE*** This hard-links the files to do this copy as fast as possible, as well as to mitigate space waste on the USB stick.
+
+   ```bash
+   rsync -rltDP --delete "${CSM_PATH}/images/kubernetes/" --link-dest="${CSM_PATH}/images/kubernetes/" "${PITDATA}/data/k8s/${kubernetes_version}"
+   rsync -rltDP --delete "${CSM_PATH}/images/storage-ceph/" --link-dest="${CSM_PATH}/images/storage-ceph/" "${PITDATA}/data/ceph/${ceph_version}"
+   ```
+
+1. (`pit#`) Generate SSH Keys and invoke `ncn-image-modification.sh`:
+
+   - Generate SSH Keys:
+
+       > ***NOTE*** The code block below assumes an RSA key without a passphrase. If the administrator wants to customize this step they may do so
+
+       ```bash
+       ssh-keygen -N "" -t rsa
+       ```
+
+   - Run `ncn-image-modification.sh` from the CSM Tarball:
+
+       ```bash
+       export SQUASHFS_ROOT_PW_HASH="$(awk -F':' /^root:/'{print $2}' < /etc/shadow)"
```

Comment:
@heemstra , I suppose I could either remove the empty line below.. or move this line into it's own step. Thoughts?